govcd/vm_test.go

```diff
@@ -652,6 +652,48 @@ func (vcd *TestVCD) Test_AnswerVmQuestion(check *C) {
 	check.Assert(isMediaInjected(vm.VM.VirtualHardwareSection.Item), Equals, false)
 }
 
+func (vcd *TestVCD) Test_VMChangeRootDiskSize(check *C) {
+	if vcd.skipVappTests {
+		check.Skip("Skipping test because vapp was not successfully created at setup")
+	}
+	vapp := vcd.findFirstVapp()
+	existingVm, vmName := vcd.findFirstVm(vapp)
+	if vmName == "" {
+		check.Skip("skipping test because no VM is found")
+	}
+
+	currentDisk := 0
```

Comment:
This variable seems to refer to capacity, rather than a disk. Please use a name that suggests its role.